sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -920,7 +928,7 @@ def reset(self):
   def process_bundle(self, instruction_id):
     # type: (str) -> Tuple[List[beam_fn_api_pb2.DelayedBundleApplication], bool]
 
-    expected_inputs = []
+    expected_input_ops = []
```

Comment:
This needed to be renamed because the variable was reused below for a different type